nuxeo-importer-xml-parser/src/main/java/org/nuxeo/ecm/platform/importer/xml/parser/XMLImporterServiceImpl.java

```diff
@@ -547,12 +546,15 @@ protected void process(Element el) {
             if (createConf != null) {
                 String chain = createConf.getAutomationChain();
                 if (chain != null && !"".equals(chain.trim())) {
-                    OperationContext ctx = new OperationContext(session, mvelCtx);
-                    ctx.setInput(docsStack.peek());
-                    try {
+                    try (OperationContext ctx = new OperationContext(session)) {
+                        ctx.putAll(mvelCtx);
+                        ctx.setInput(docsStack.peek());
                         getAutomationService().run(ctx, chain);
-                    } catch (OperationException e) {
-                        throw new NuxeoException(e);
+                    } catch (Exception cause) {
```

Comment:
Add a call to `ExceptionUtils.checkInterrupt(cause)` first thing after the `catch (Exception...)`. Also the usage in our code base is to call the exception `e` instead of `cause` in `catch` blocks when possible.